grommet-leaflet/src/layers/Controls/index.d.ts

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+
+export interface ControlsProps {
+  locations?: number[];
```

Comment:
Should this be more specific as an array of [latitude, longitude]? Leaflet might provide a type for that.

depending on what that is, something like
```
locations?: LatLng[]
```

Reply:
Good suggestion, leaflet does provide a type